_posts/convars/2022-02-02-mom_df_physics.md

```diff
@@ -0,0 +1,13 @@
+---
+title: mom_df_physics
+category: var
+tags:
+  - defrag
+  - physics
```

Comment:
Physics is a bit confusing of a term, it's really changing the type of playermovement; best to stick to the movement & player tags and remove this physics one.